### ISSUE TITLE
On-boarding

### ISSUE THREAD

**Issue by franticpsyx** [Prospress/woocommerce-subscribe-all-the-things#314]
NUX is important.

I'm not sure if we should somehow borrow some design language/elements from the WCS on-boarding process and frankly haven't even looked at what the WCS on-boarding process looks like -- is there one, even?

What users need to know, in short:

* How to create product-level subscription schemes/options.
* How to offer cart subscriptions.
* How to allow customers to add products to existing subscriptions.

@thenbrent we might also want to look into ways to give SATT a more prominent position in the WCS on-boarding/welcome process: As a store owner, I would like to know the pros/cons of working with subscription product types vs subscription schemes. Might (also) be worth looking at creating a WCS doc? In any case, I'd appreciate any input you can provide here. If you feel this shouldn't be given priority, lmk.

**Comment by thenbrent** [comment]
> haven't even looked at what the WCS on-boarding process looks like

Yes. There's three elements to it:

1. unobtrusive admin notice saying welcome to Subscriptions
2. JS popups to guide someone through creating a subscription product - this happens _only_ if the "Add subscription product" button is clicked in the above notice
3. screen prompting user to add a subscription product when visiting the **WooCommerce > Subscriptions** page, but not having any subscriptions (this is because some folks thought that's where they'd add subscription _products_).

To see the first two, I think you can delete the `'woocommerce_subscriptions_is_active'` row form the options table then load an admin page.

**Comment by thenbrent** [comment]
> I think you already did something similar in the WCS 2.1 release?

Yeah, we had a section for it in there. Also after that "_  and..._" section I included above but didn't finish, I was going to say "_we're thinking about phasing those out now_". That hasn't been decided yet. If we keep them, SATT should definitely be featured in it.

> reserve some space there for SATT features released within the current WCS cycle

I really like that. Keeps it relevant and fresh, rather than just saying "Oh, here's SATT again...".

> We could show a "did you know" message later on

My concern with that is that I know some folks come to WCS wanting to do at the outset exactly some of the things SATT provides (namely, allow them to set Subscriptions Schemes on a one-off product).

I do really like the idea of building out some "progressive education" in WCS though, to provide "did you knows" for a variety of more complex features, so the learning doesn't stop after the on-boarding, and SATT would fit nicely into that.

> I'll let this sit for a while and apply that experience here.

Let me know if you have any issues using seeing the WCS flow (i.e. the JS pop-ups). If so, I'll run through it and do a quick recording.

**Comment by thenbrent** [comment]
> we're thinking about phasing those out now". That hasn't been decided yet

For reference, here is the new approach being proposed for 2.3: https://github.com/Prospress/woocommerce-subscriptions/pull/2622 (I forgot we already had a PR up for that).
